server/src/test/java/org/elasticsearch/cluster/routing/DelayedAllocationServiceTests.java

```diff
@@ -167,7 +168,8 @@ public void testDelayedUnassignedScheduleReroute() throws Exception {
             clusterStateUpdateTask.set((ClusterStateUpdateTask) invocationOnMock.getArguments()[1]);
             latch.countDown();
             return null;
-        }).when(clusterService).submitStateUpdateTask(eq(CLUSTER_UPDATE_TASK_SOURCE), any(ClusterStateUpdateTask.class));
+        }).when(clusterService)
+            .submitStateUpdateTask(eq(CLUSTER_UPDATE_TASK_SOURCE), any(ClusterStateUpdateTask.class), ClusterStateTaskExecutor.unbatched());
```

Comment:
Here and in other places of this file it should be `any()` instead of `unbatched` I believe